### DISCUSSION TITLE
„Error: Missing parameter name at 5“ during deployment

### DISCUSSION THREAD

**Discussion by johannesbraeunig** [vercel/community#1594] [Help] [answered]
Hi, since 2 days I get this error after the successful build of a remix app. This error message says nothing to me, as this does not seem to be an issue with the app itself.

Even successful deployments from before 2 days I tried to redeploy are failing with the same error now.

Does anyone experience the same or even better, has anyone some advice where I should start looking into to fix it?

![image](https://user-images.githubusercontent.com/1436744/220434378-11e2185a-49fb-43d2-b689-0acbd2bd4931.png)

Thanks! 🙏

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out, Johannes. Can you temporarily add an environment variable named `VERCEL_CLI_VERSION` set to `vercel@28.15.3` and try again? This will help determine if the problem is related to the CLI or if we need to look at another possible cause

**Reply by johannesbraeunig** [reply]
@amyegan oh wow, that was it! Thanks a bunch! I didn't know that env-variable is required, should it be required?

**Reply by amyegan** [reply]
It shouldn't be required normally. That's just what we use to test whether the CLI version is disagreeing with your code. It's only a temporary solution which, thankfully, got you unstuck for now. But we still need to figure out what caused the problem. 

Do you happen to have a public repo, or a minimal reproduction, that I can share with the engineering team to help us with debugging? Any reproduction details you can share would be very much appreciated 🙏
